lib/core/macros.nim

```diff
@@ -281,9 +281,12 @@ else: # bootstrapping substitute
 
 {.pop.}
 
-when defined(nimSymImplTransform):
+since (1,3,5):
```

Comment:
This doesn't make sense and is a breaking change.

Reply:
> breaking change

only if passing `--useVersion:1.0 -d:nimSymImplTransform` and passing `--useVersion:1.0` is for testing not production use; anyhow I've now changed this